platforms/software/test-suites-base/src/main/java/org/gradle/testing/base/plugins/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Plugin classes for generic support for testing.
+ */
+package org.gradle.testing.base.plugins;
```

Comment:
❓ Shouldn't this be failing without an `org.gradle.api.NonNullApi` annotation per Alex's recent change?

Reply:
Do you mean the unmerged https://github.com/gradle/gradle/pull/26593 ?

Reply:
Yes.  My mistake, I assumed it was merged by now.  I think it's still worth annotating the package in advance.

Reply:
I did so.